main/RadioButton/index.tsx

```diff
@@ -56,12 +56,9 @@ const SCInputRow = styled.TouchableOpacity<IInputRowProps>`
   padding: 8px 0;
 `;
 
-const SCOuterCircle = styled.View.attrs(
-  ({ borderRadius, borderWidth }: ICircleProps) => ({
-    borderRadius,
-    borderWidth,
-  }),
-)<ICircleProps>`
+const SCOuterCircle = styled.View<ICircleProps>`
```

Comment:
`SCOuterCircle` should be changed for better readable.
The same is `SCInputRow`.
I think there's need to re-define of `SC`.

Reply:
I'll consider it. :)